.github/workflows/on-pull-request.yml

```diff
@@ -4,7 +4,9 @@ on:
   pull_request:
     branches:
       - main
-
+    paths:
+      - "modules/**"
```

Comment:
Need to remove this. The CI job also checks style for non-module files.

Reply:
Another reason is that if the workflow is a required check. Pull requests that do not include changes to modules cannot be merged with the path filtering. See: https://github.community/t/path-filtering-on-required-pull-request-checks/18402.